jsclient/routes/SimpleWalletClient.js

```diff
@@ -143,6 +143,8 @@ class SimpleWalletClient {
 		   	.then((responseJson) => {
                         	var data = responseJson.data;
                         	var amount = new Buffer(data, 'base64').toString();
+				if(amount == "")
+                                        amount = 0;
```

Comment:
indentation should be just 4 spaces.
Someday, we'll have to fix up the entire code indentation. :)

Reply:
Could we remove this condition as overriding empty value with 0 would be misleading that the balance has become 0.